config/rbac/leader_election_role_binding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
all files in rbac folder should use the default namespace (if the have namespace of course, like service account & role binding) and not the system namespace as it set now. we will change the namespace in our values folder but for now the default namespace is the base line